dotenv-sample

```diff
@@ -4,7 +4,12 @@ ADMIN_USERS=
 
 # Note: this assumes docker dev db. You will need to change it if you want to
 # point another postgres
-DATABASE_URL=postgres://user:pass@localhost:6534/jobserver
+DATABASE_USER="user"
+DATABASE_PASSWORD="pass"
+DATABASE_DB="jobserver"
+DATABASE_HOST="localhost"
+DATABASE_PORT="6543"
+DATABASE_URL=postgres://${DATABASE_USER}:${DATABASE_PASSWORD}@${DATABASE_HOST}:${DATABASE_PORT}/${DATABASE_DB}
```

Comment:
Rather than split these out into separate environments, we could easily parse them out when we need them for this commnad?

urllib.parse.urlparse does this fine e.g.

```
from urllib.parse import urlparse

db = urlparse(settings.DATABASE_URL)
# db.username, db.password, db.hostname, db.port, db.path, etc
```

psycopg2 also has [Connection.dsn_parameters](https://www.psycopg.org/docs/extensions.html#psycopg2.extensions.ConnectionInfo.dsn_parameters), but it looks like it doesn't include the password.